todo_app/tests/test_app.py

```diff
@@ -0,0 +1,131 @@
+import os
+import pytest
+import requests
+from todo_app import app
+from dotenv import load_dotenv, find_dotenv
+
+@pytest.fixture
+def client():
+    # Use our test integration config instead of the 'real' version
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+
+    # Create the new app.
+    test_app = app.create_app()
+
+    # Use the app to create a test_client that can be used in our tests.
+    with test_app.test_client() as client:
+        yield client
+
+class StubResponse():
+    def __init__(self, fake_response_data):
+        self.fake_response_data = fake_response_data
+
+    def json(self):
+        return self.fake_response_data
+
+# Stub replacement for requests.get(url)
+def stub(url, params={}):
+    test_board_id = os.environ.get('CORNDEL_BOARD_ID')
+    test_card_id = "123abc"    
+    trello_key = os.environ.get('TRELLO_API_KEY')
+    trello_token = os.environ.get('TRELLO_TOKEN')
+
+    if url == f'https://api.trello.com/1/boards/{test_board_id}/lists':
+        fake_response_data = [{
+            'id': '123abc',
```

Comment:
Minor,  but given you've made an effort to extract this id into a single variable, would it be worth using that everywhere to save you having mismatching ids when someone inevitably changes this?
```suggestion
            'id': test_card_id,
```